pkg/nsx/services/store.go

```diff
@@ -25,32 +25,28 @@ var (
 func securityPolicyCRUIDScopeIndexFunc(obj interface{}) ([]string, error) {
 	// TODO: simplify this function
 	res := make([]string, 0, 5)
-	switch obj.(type) {
```

Comment:
Could you help remove the `TODO` in L26? thanks!

Reply:
done